artifact/sha/platform/openssl/sha.cpp

```diff
@@ -78,9 +78,9 @@ Reader::Reader(io::Reader &reader, const std::string &expected_sha = "") :
 expected::ExpectedSize Reader::Read(
 	vector<uint8_t>::iterator start, vector<uint8_t>::iterator end) {
 	if (!initialized_) {
-		return MakeError(
+		return expected::make_unexpected(MakeError(
```

Comment:
As I asked in the other PR -- could we do `using unexpected = make_unexpected;` or something like that? `std::expected` doesn't have `make_unexpected()`, right?

Reply:
I reckon we can follow the same rules as everywhere else: We can make such statements in any local namespace/source, but let's keep it out of the global scope. Whether we should do it in this mass replacement.. I don't have a strong opinion, either is fine with me. It does mean it would need to be added multiple times.

Reply:
We can do this:

```
#ifdef __cpp_lib_expected
using std::expected;
using std::unexpected;
#else
using tl::expected;
template <typename V>
tl::unexpected<V> unexpected(V &&v) {
	return tl::unexpected<V>(v);
}
#endif
```

Which maps nicely to the std-lib then

Reply:
👍

Reply:
Done!

Reply:
:clap:  thanks!